spanner/spansql/parser.go

```diff
@@ -1832,6 +1832,23 @@ func (p *parser) parseSelect() (Select, *parseError) {
 		}
 
 		for {
+			if len(sel.From) > 0 {
+				var lhs SelectFrom = SelectFromList(sel.From)
+				if len(sel.From) == 1 {
+					lhs = sel.From[0]
+				}
+				sfj, err := p.parseJoin(lhs)
```

Comment:
Does this work if the query contains from table after a join? So something like this:
```sql
SELECT *
FROM Table1
INNER JOIN Table2 ON Table1.Id=Table2.Id, SomeOtherTable
```